toolkit/docs/building/building.md

```diff
@@ -1,25 +1,24 @@
 # Building
-
-- [Building](#building)
-  - [Quick Start](#quick-start)
-    - [Prerequisites](#prerequisites)
-    - [Configure the Environment](#configure-the-environment)
-    - [ISO Quick Start](#iso-quick-start)
-    - [VHDX Quick Start](#vhdx-quick-start)
-    - [Build from Sources Quick Start](#build-from-sources-quick-start)
-  - [Further Reading](#further-reading)
-    - [Building in Stages](#building-in-stages)
-      - [1) Bootstrap Toolchain](#1-bootstrap-toolchain)
-        - [Full toolchain Bootstrap From Sources](#full-toolchain-bootstrap-from-sources)
-      - [2) Build All Packages](#2-build-all-packages)
-      - [3) Build Images](#3-build-images)
-    - [ISOs](#isos)
-    - [Packages](#packages)
-      - [Working on Packages](#working-on-packages)
-        - [DOWNLOAD_SRPMS](#download_srpms)
-        - [Force Rebuilds](#force-rebuilds)
-        - [Ignoring Packages](#ignoring-packages)
-        - [Source Hashes](#source-hashes)
+- [Overview](#Overview)
```

Comment:
>Overview [](start = 14, length = 8)

It seems the overview link doesn't work. At least on GitHub's PR rich preview. Can you try using small-caps?

Reply:
Pass on the small caps...not used anywhere else, but I did lowercase the O in the link.  It did and still does work locally.

Reply:
I guess I should have been more precise: the link is what I had in mind, so it looks good to me and the GitHub rich preview now understands the link - maybe it's not a problem when viewed differently. For the actual text I'd stay with capital letters as well.

---
In reply to: [546956144](https://github.com/microsoft/CBL-Mariner/pull/476#discussion_r546956144) [](ancestors = 546956144)